build.gradle

```diff
@@ -0,0 +1,54 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.70"
+    }
+}
+
+plugins {
+    id 'java'
+    id 'org.jetbrains.intellij' version '0.4.16'
+    id "org.jetbrains.kotlin.jvm" version "1.3.70"
+}
+
+repositories {
+    mavenCentral()
+    jcenter()
+}
+
+allprojects {
+    apply plugin: 'idea'
+    idea {
+        module {
+            downloadSources = true
+            downloadJavadoc = true
+        }
+    }
+}
+
+sourceCompatibility = 1.8
+
+apply plugin: 'java'
+apply plugin: 'kotlin'
+apply plugin: 'org.jetbrains.intellij'
+
+intellij {
+    version '2019.3.3'
+    plugins = ['Kotlin', 'java']
+    pluginName 'KotlinCodeSmellDetector'
+    downloadSources true
+    intellij.updateSinceUntilBuild false
+}
+
+dependencies {
+    compile group: 'org.eclipse.mylyn.github', name: 'org.eclipse.egit.github.core', version: '2.1.5'
+}
+
+jar {
+    from {
+        configurations.runtimeClasspath.collect { it.isDirectory() ? it : zipTree(it) }
```

Comment:
Do we need this?

Reply:
Don't know, I took it from IntelliJDeodorant. Removed for now.